src/messages.json

```diff
@@ -104,11 +104,13 @@
         "ProgramBlockEditor.command": "{command}, position {index} of current program",
         "ProgramBlockEditor.program.deleteAll": "Clear and start new program",
         "ProgramBlockEditor.programHeading": "Program",
-        "ProgramBlockEditor.spaceShipCharacter": "Space Ship character at column {xPos}, row {yPos} facing {direction}",
-        "ProgramBlockEditor.rabbitCharacter": "Rabbit character at column {xPos}, row {yPos} facing {direction}",
-        "ProgramBlockEditor.robotCharacter": "Robot character at column {xPos}, row {yPos} facing {direction}",
-        "ProgramBlockEditor.editPosition.trunLeft": "Turn the character left",
-        "ProgramBlockEditor.editPosition.trunRight": "Turn the character right",
+        "ProgramBlockEditor.movementAriaLabel": "{character} is moving",
+        "ProgramBlockEditor.positionAriaLabel": "{character} at column {xPos}, row {yPos} facing {direction}",
+        "ProgramBlockEditor.spaceShipCharacter": "Space Ship character",
+        "ProgramBlockEditor.rabbitCharacter": "Rabbit character",
+        "ProgramBlockEditor.robotCharacter": "Robot character",
```

Comment:
This is something that I didn't catch when we moved the ARIA live region code to the `CharacterAriaLive` component, but these messages should begin with "CharacterAriaLive" rather than "ProgramBlockEditor". Could we make that fix in this PR please?

Reply:
Done.